src/module.js

```diff
@@ -1195,6 +1197,31 @@ export default class DailyIframe extends EventEmitter {
     return this;
   }
 
+  showLocalVideo(bool = true) {
+    methodNotSupportedInReactNative();
```

Comment:
should this check also be in `showParticipantsBar()`?

Reply:
these methods only apply to embedded prebuilt UI mode, right? if so, we could bail early and print a console error if you invoke them in call object mode (if `this._callObjectMode`). we're not very consistent about doing them across the board, but better late than never right?

Reply:
Yes to both, great point!